security/README.md

```diff
@@ -0,0 +1,42 @@
+Security
+========
+
+A guide for practicing safe web.
+
+Think
+-----
+
+Security is important, and you can't practice these guidelines without
+understanding them. Question each guideline before following it yourself and
+start a discussion whenever you're in doubt.
+
+Passwords
+---------
+
+* Use a unique password for every account you create.
+* Don't store passwords in plain text.
```

Comment:
More than not storing it in plain text: do not hash it using a reversible cipher, do not encrypt it using a broken cipher (e.g. SHA1).